### PR TITLE
Image component

### PR DESCRIPTION
**Pull request by prath-dev**
Image Component Added .
Live Preview : http://127.0.0.1:5501/index.html#image

### CODE REVIEW

**Review by jeetbhalani15 — APPROVED** [review]
good to go

File: components/Images/images.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+.round img {
+    height: 12rem;
+    width: 12rem;
+    border-radius: 13rem;
+}
+
+.inputs-container {
+    margin-bottom: 1rem;
+    height: 8rem;
+    width: 35rem;
```
Comment (anurag-kkumarr) [inline components/Images/images.css:31]: Use variable for colors and sizes.

**Review by anurag-kkumarr — CHANGES_REQUESTED** [review]
Else, It looks fine

**Review by Vinayak-YB — APPROVED** [review]
The code looks good to me.

**Comment by prath-dev** [comment]
@anurag-kkumarr  : i will do  changes in second version of component

**Pull request merged**